### ISSUE TITLE
win版有自定义源管理希望安卓版也加入这个功能

### ISSUE THREAD

**Issue by yanshixin3** [lyswhut/lx-music-mobile#347]
### 解决方案检查

- [X] 我已阅读常见问题(<https://lyswhut.github.io/lx-music-doc/mobile/faq>)，但没有找到解决方案
- [X] 我已搜索issue列表(<https://github.com/lyswhut/lx-music-mobile/issues?utf8=✓&q=>)，但没有发现类似的问题

### 问题描述

win版有自定义源管理希望安卓版也加入这个功能

### 描述你想要的解决方案

win版有自定义源管理希望安卓版也加入这个功能

### 描述你考虑过的替代方案

win版有自定义源管理希望安卓版也加入这个功能

### 附加信息

win版有自定义源管理希望安卓版也加入这个功能

**Comment by dmh91168** [comment]
同意添加

**Comment by ameaninglessname** [comment]
#329 重复issue

**Comment by CUMTBBolei** [comment]
Agreed, I'm proposing to add the ability to customize the source in the mobile version (Android).

**Comment by hxy310** [comment]
附议！附议！，或者怎么用mt解包修改也行啊。

**Comment by q8576** [comment]
附意

**Comment by nkxingxh** [comment]
![image](https://github.com/lyswhut/lx-music-mobile/assets/25559053/e792d807-2090-40af-94bc-1db4427bff9f)

**Comment by hxy310** [comment]
> ![图像](https://user-images.githubusercontent.com/25559053/279946239-e792d807-2090-40af-94bc-1db4427bff9f.png)

怎么弄的啊，能不能教教。

**Comment by nkxingxh** [comment]
自行编写音源，参考洛雪内置音源相关代码即可。有能力的话也可以移植桌面端的自定义音源JS。

**Comment by congmoyue** [comment]
> 自行编写音源，参考洛雪内置音源相关代码即可。有能力的话也可以移植桌面端的自定义音源JS。

大佬，体谅一下这些小白确实不会弄这个操作能否放出一个包，或者是写个教程也行

**Comment by nkxingxh** [comment]
> > 自行编写音源，参考洛雪内置音源相关代码即可。有能力的话也可以移植桌面端的自定义音源JS。
> 
> 大佬，体谅一下这些小白确实不会弄这个操作能否放出一个包，或者是写个教程也行

你看得懂js代码就完全可以参考洛雪源代码和桌面端自定义音源编写教程了。如果不懂js我出教程也没用，总结就是聊胜于无。

你要成品的话倒是可以看这个 【洛雪音乐手机版可用方案-哔哩哔哩】 https://b23.tv/gXXI1sj。

**Comment by binggz** [comment]
通过与pc的同步实现的话，将非常方便

**Comment by wluckcn** [comment]
同求啊，目前所有接口都不能用了

**Comment by lyswhut** [comment]
v1.2.0-beta新增了自定义源的功能（实验性），
可以去这里下载测试：https://github.com/lyswhut/lx-music-mobile/actions/workflows/beta-pack.yml
该版本更新日志看：https://github.com/lyswhut/lx-music-mobile/blob/beta/publish/changeLog.md

**Comment by huang-boom** [comment]
> v1.2.0-beta新增了自定义源的功能（实验性）， 可以去这里下载测试：https://github.com/lyswhut/lx-music-mobile/actions/workflows/beta-pack.yml 该版本更新日志看：https://github.com/lyswhut/lx-music-mobile/blob/beta/publish/changeLog.md

感谢落雪大佬，手机端终于再次实现听歌自由了👍👍👍

**Comment by chuxnj** [comment]
> > v1.2.0-beta增加了自定义源的功能（实验性），可以去这里下载测试：https://github.com/lyswhut/lx-music-mobile/actions/workflows/beta-pack.yml该版本更新日志看：https://github.com/lyswhut/lx-music-mobile/blob/beta/publish/changeLog.md
> > **注意，此功能目前仍处于测试阶段，请勿到处宣传！**
> > 感谢大佬

你们的都是构建打包吗？

**Comment by chuxnj** [comment]
> > v1.2.0-beta增加了自定义源的功能（实验性），可以去这里下载测试：https://github.com/lyswhut/lx-music-mobile/actions/workflows/beta-pack.yml该版本支付日志看：https://github.com/lyswhut/lx-music-mobile/blob/beta/publish/changeLog.md
> > **注意，此功能目前尚处于测试阶段，请勿到处宣传！**
> > 感谢大佬

你们都是自己构建打包吗

**Comment by pencil1985** [comment]
导入音源成功，感谢作者。

**Comment by MysteriousNullPoint** [comment]
导入成功,非常感谢

**Comment by Jimmylxue** [comment]
真的感谢作者！

**Comment by Xuhao9527** [comment]
> > v1.2.0-beta新增了自定义源的功能（实验性）， 可以去这里下载测试：https://github.com/lyswhut/lx-music-mobile/actions/workflows/beta-pack.yml 该版本更新日志看：https://github.com/lyswhut/lx-music-mobile/blob/beta/publish/changeLog.md
> 
> 感谢落雪大佬，手机端终于再次实现听歌自由了👍👍👍

大佬怎么下载使用呀

**Comment by ameaninglessname** [comment]
> > > v1.2.0-beta新增了自定义源的功能（实验性）， 可以去这里下载测试：https://github.com/lyswhut/lx-music-mobile/actions/workflows/beta-pack.yml 该版本更新日志看：https://github.com/lyswhut/lx-music-mobile/blob/beta/publish/changeLog.md
> > 
> > 
> > 感谢落雪大佬，手机端终于再次实现听歌自由了👍👍👍
> 
> 大佬怎么下载使用呀

自己多搜索下：
https://github.com/lyswhut/lx-music-mobile/actions/runs/6759890062

**Comment by lin588-ui** [comment]
隔壁助手关闭登录了，配合那边的源使用挺不错。感谢作者大大。

**Comment by xian410** [comment]
感谢作者！！

**Comment by dovsay** [comment]
可以用了，感谢作者。

**Comment by CN-AC** [comment]
感谢作者！！

**Comment by jayhuang861019** [comment]
怎么添加源呀  哪里有源

**Comment by Muyan2** [comment]
> v1.2.0-beta新增了自定义源的功能（实验性）， 可以去这里下载测试：https://github.com/lyswhut/lx-music-mobile/actions/workflows/beta-pack.yml 该版本更新日志看：https://github.com/lyswhut/lx-music-mobile/blob/beta/publish/changeLog.md
> 
> 提示：Actions的文件需要登录GitHub才可以下载。 **注意，这个功能目前仍然处于测试阶段，请勿到处宣传！**

感谢大佬！！！！可以用了！

**Comment by wj2008** [comment]
希望作者内置音源能否添加一个在线地址的功能，或者有没有大佬告知一下怎么改。

**Comment by HAIGENSHUO** [comment]
安卓版不是早就有了

wj2008 ***@***.***> 于 2023年12月19日周二 上午11:06写道：

**Comment by qq656380143** [comment]
有大佬写音源出来吗？六音不能下载了，坏了估计也不会修复了，也不知道能用多久
